torch/csrc/jit/pybind_utils.h

```diff
@@ -543,6 +544,10 @@ inline IValue toIValue(
     case TypeKind::FutureType:
     case TypeKind::QSchemeType:
       break;
+    case TypeKind::PyObjectType:
+      // convert a pybind11 object to the IValue that holds the
+      // PyObject and steal the reference to take ownership of it.
+      return c10::ivalue::ConcretePyObjectHolder::steal(obj.ptr());
```

Comment:
This steals the reference meaning somewhere else in the code a decref needs to _not_ happen.  But I see no code that would accomplish this. Should this not borrow the reference (i.e incref the object before handing to PyObjectHolder)